src/shared/components/mutationTable/MutationTable.tsx

```diff
@@ -658,6 +660,23 @@ export default class MutationTable<
                     .indexOf(filterStringUpper) > -1,
         };
 
+        this._columns[MutationTableColumnType.VARIANT_TYPE] = {
+            name: 'Variant Type',
+            render: VariantTypeColumnFormatter.renderFunction,
+            download: VariantTypeColumnFormatter.getTextValue,
+            sortBy: (d: Mutation[]) =>
+                VariantTypeColumnFormatter.getDisplayValue(d),
+            filter: (
+                d: Mutation[],
+                filterString: string,
+                filterStringUpper: string
+            ) =>
+                VariantTypeColumnFormatter.getDisplayValue(d)
```

Comment:
instead of indexOf, use  .includes() which returns boolean.

Reply:
@alisman general question:  do people usually address code review feedback in a new commit?  and, after the reviewer has re-reviewed, I then squash all the commits into one?

Reply:
ok, found 6 instances of `indexOf()` within `MutationTable.tsx`.  Changed all of these to `includes()`.